### DISCUSSION TITLE
Is it possible to run some custom code in Supabase?

### DISCUSSION THREAD

**Discussion by Empirome** [supabase/supabase#17724] [Questions]
Hi there!

In one of my projects, I have two tables which I need to compare some data of these tables. As a result of these comparisons I would need to some updates, deletions & inserts into a third table. These comparisons need to run through some loops, like for...next and while...wend.

Is this even possible to do in Supabase? I'm thinking about something similar to VBA in Microsoft Access where you open the recordset and can edit it based on some conditions.

Couldn't find anything googling for it so your help would be much appreciated.

Thank you very much already!

**Comment by sergchernata** [comment]
You should be able to do that with [edge functions](https://supabase.com/docs/guides/functions) or even [psql triggers / functions](https://supabase.com/docs/guides/database/postgres/triggers).
